drivers/gles3/rasterizer_storage_gles3.h

```diff
@@ -1199,14 +1193,14 @@ class RasterizerStorageGLES3 : public RendererStorage {
 			GLuint color = 0;
 			GLuint depth = 0;
 			RID texture;
-
-			External() {
-			}
 		} external;
 
-		int x = 0, y = 0, width = 0, height = 0;
+		int x = 0;
+		int y = 0;
+		int width = 0;
+		int height = 0;
 
-		bool flags[RENDER_TARGET_FLAG_MAX];
+		bool flags[RENDER_TARGET_FLAG_MAX] = {};
```

Comment:
This was just for good measure, uninitialized flags are a good way to get random visual errors.